.travis.yml

```diff
@@ -1,5 +1,24 @@
+services:
+    - docker
+before_script:
+    - docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+    - curl https://cli-assets.heroku.com/install.sh | sh  #install heroku
+    - docker login --username=$HEROKU_USER --password=$HEROKU_API_KEY registry.heroku.com
 script:
     - docker build --target test --tag my-test-image .
     - docker run my-test-image tests/unit
     - docker run my-test-image tests/integration
-    - docker run --env apiKey --env apiToken --env boardId --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_APP --env FLASK_ENV my-test-image tests/endtoend
\ No newline at end of file
+    - docker run --env apiKey --env apiToken --env boardId --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_ENV my-test-image tests/endtoend
+    - docker build --target production --tag $DOCKER_USER/todo-app:latest .  
+    - docker push $DOCKER_USER/todo-app:latest
```

Comment:
It's worth noting that you push this image on every pull request - in general you'd likely want to only do that on merge to master (i.e. when you're deploying) in case a review flags concerns. 
This might be a good place to take advantage of the `before_deploy` travis lifecycle stage ([mentioned in the docs here](https://docs.travis-ci.com/user/job-lifecycle/#deploying-your-code))

Reply:
Changed branch to master for deploy